server/taboo.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const TabooWordsSchema = new Schema({
+  word: {type: String, required: true},
+  suggesterId: {type: Number, ref: 'User'},
```

Comment:
The type for a foreign key should be type: Schema.Types.ObjectId

Reply:
corrected type